data/human/far north jobs.txt

```diff
@@ -90,9 +90,11 @@ mission "Navy resupply [0]"
 	to offer
 		random < 2
 	source
+		government "Republic" "Syndicate"
 		not government Pirate
```

Comment:
```suggestion
```
If the government has to be Republic or Syndicate, it can't be Pirate.

Reply:
Right.